tests/_runtime/test_runtime.py

```diff
@@ -32,11 +32,11 @@
     from types import ModuleType
 
 
-def _check_edges(error: Error, expected_edges: Sequence[Edge]) -> None:
+def _check_edges(error: Error, expected_edges: Sequence[EdgeWithVar]) -> None:
     assert isinstance(error, CycleError)
-    assert len(error.edges) == len(expected_edges)
+    assert len(error.edges_with_vars) == len(expected_edges)
     for edge in expected_edges:
-        assert edge in error.edges or (edge[1], edge[0]) in error.edges
+        assert edge in error.edges_with_vars or reversed(edge) in error.edges_with_vars
```

Comment:
I believe the reversed logic wasn't needed originally; removed it and these tests
still pass on my machine. Checking for the presence of a reversed edge is no longer
correct, since the variables that flow on the edges change when you flip the direction.

```suggestion
        assert edge in error.edges_with_vars
```